src/Library/demos/TextView/main.js

```diff
@@ -0,0 +1,63 @@
+  -1,
+);
+
+textview.buffer = buffer;
+
+function handleCopy() {
+  if (buffer.get_has_selection()) {
+    buffer.copy_clipboard(clipboard);
+    console.log("Text copied to clipboard");
+  } else {
+    console.log("No text selected to copy");
+  }
+}
+
+function handlePaste() {
+  buffer.paste_clipboard(clipboard, null, true);
+  console.log("Text pasted");
+}
+
+function handleCut() {
+  if (buffer.get_has_selection()) {
+    buffer.cut_clipboard(clipboard, true);
+    console.log("Text cut to clipboard");
+  } else {
+    console.log("No text selected to cut");
+  }
+}
+
+copy.connect("clicked", handleCopy);
+paste.connect("clicked", handlePaste);
+cut.connect("clicked", handleCut);
```

Comment:
This is great with the buttons being tied to the buffer. I was thinking the better way is to use the (undocumented) GActions, like `clipboard.copy`, but then I think those would be less obvious being in the UI file.

What do you think? If you think it's fine like this, let's go ahead and we can revisit later if necessary.

Reply:
Maybe we could comment the extra methods?

Reply:
Probably it would be best done with gi-docgen annotations (used to generate docs.gtk.org etc), because they can add extra things that aren't a part of GI. That's a longer fix, but I'm not sure GActions are guaranteed to be "API-stable", at least not until they're part of the documentation.

Maybe we can bring this up in #321, or a related demo.